src/evm/contracts/ILightClient.sol/IMembershipVerifier.ts

```
@@ -0,0 +1,135 @@
+    "nonpayable"
+  >;
+
+  getFunction<T extends ContractMethod = ContractMethod>(
+    key: string | FunctionFragment
+  ): T;
+
+  getFunction(
+    nameOrSignature: "verifyMembership"
+  ): TypedContractMethod<
+    [proof: Ics23ProofStruct, key: BytesLike, expectedValue: BytesLike],
+    [void],
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)